src/lib/frontend/core/ToasterUtils.ts

```diff
@@ -1,25 +1,15 @@
-import toast from 'svelte-french-toast';
+import type {ToastSettings} from '@skeletonlabs/skeleton'
 
-export function toastError(message: string): string {
-	return toast.error(message, {
-		style: 'background-color: #EF4444; color: white',
-		iconTheme: {
-			primary: '#fff',
-			secondary: 'red'
-		}
-	});
+export function getErrorToastSettings(message: string): ToastSettings {
+    return {
+        message,
+        background: 'variant-filled-error'
+    } satisfies ToastSettings
 }
 
-export function toastGenericError(): void {
-	toastError("Désolé, quelque chose n'a pas fonctionné");
-}
-
-export function toastSuccess(message: string): void {
-	toast.success(message, {
-		style: 'background-color: #F0FDF4;',
-		iconTheme: {
-			primary: 'green',
-			secondary: 'white'
-		}
-	});
+export function getSuccessToastSettings(message: string): ToastSettings {
```

Comment:
Is it possible to start now to put every configuration of the look and feel in one file at the root level of the project? And then use them every where else thanks to the approach we have discussed (meta.env it was I believe)

Reply:
I saw in the recorded video of this morning meeting that Adderly was already working on the project configuration, maybe we can let him finish the PR, so depending of how it is done.  I will see how the import will be made

Reply:
sounds good